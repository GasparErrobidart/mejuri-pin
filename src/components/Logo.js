import React    from 'react'
import styled   from 'styled-components'

const LogoWrapper = styled.div`
  height: 50px;
  padding: 5px 0;
  & > img{
    height: 100%;
    width: auto;
  }
`;

const Logo = ()=>{
  return(
    <LogoWrapper>
      <img src="/mejuri.svg" alt="Mejuri Logo"/>
    </LogoWrapper>
  )
}

export default Logo;
