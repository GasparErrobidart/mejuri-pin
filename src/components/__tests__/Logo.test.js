import '@testing-library/jest-dom'
import 'jest-styled-components'

import React         from 'react'
import { cleanup }   from '@testing-library/react'
import renderer      from 'react-test-renderer'

import Logo   from '../Logo'


afterEach(cleanup)

test('Renders "Logo" component', () => {
  const testText        = "Logo Test Text"
  const component       = renderer.create(<Logo/>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
