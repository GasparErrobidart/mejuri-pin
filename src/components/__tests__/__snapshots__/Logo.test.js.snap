// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "Logo" component 1`] = `
.c0 {
  height: 50px;
  padding: 5px 0;
}

.c0 > img {
  height: 100%;
  width: auto;
}

<div
  className="c0"
>
  <img
    alt="Mejuri Logo"
    src="/mejuri.svg"
  />
</div>
`;
